mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/processing/impl/IndexedProcessor.java

```diff
@@ -41,7 +42,16 @@ public void process(TypeMappingStep mappingContext, Indexed annotation,
 				routingBinderReferenceAnnotation.type(), routingBinderReferenceAnnotation.name(),
 				routingBinderReferenceAnnotation.retrieval()
 		);
-		if ( routingBinderReference.isPresent() ) {
+
+		if ( !routingBinderReference.isPresent() ) {
+			return;
+		}
+
+		if ( routingBinderReferenceAnnotation.params() != null ) {
+			Map<String, Object> params = ParameterAnnotationsReader.toMap( routingBinderReferenceAnnotation.params() );
```

Comment:
What's the point of exposing a method in the `context` if you don't use it? :)

```suggestion
			Map<String, Object> params = context.toMap( routingBinderReferenceAnnotation.params() );
```